hack/python-sdk/gen-sdk.sh

```diff
@@ -18,14 +18,12 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-repo_root="$(dirname "${BASH_SOURCE}")/../.."
```

Comment:
After changing, this I can see that it creates invalid folders: https://github.com/kubeflow/training-operator/tree/f8b8951c1ef6fd853c9c3db3e40ddba949e546c2/hack/python-sdk